src/rpc/blockchain.cpp

```diff
@@ -359,17 +359,23 @@ UniValue getdifficulty(const JSONRPCRequest& request)
 std::string EntryDescriptionString()
 {
     return "    \"size\" : n,             (numeric) virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted.\n"
-           "    \"fee\" : n,              (numeric) transaction fee in " + CURRENCY_UNIT + "\n"
-           "    \"modifiedfee\" : n,      (numeric) transaction fee with fee deltas used for mining priority\n"
+           "    \"fee\" : n,              (numeric) transaction fee in " + CURRENCY_UNIT + " (DEPRECATED) \n"
```

Comment:
Nit, remove space between `)` and `\n`. Same below.

Reply:
This also affects `getmempoolancestors`, `getmempooldescendants` and `getmempoolentry`. Update respective tests and release notes?

Reply:
I'm not sure what needs to be updated in the tests for the above RPC commands as there's an `assert_equal` check against `mempool` dict for all 3 RPC commands.